lib/hemp/controller.rb

```diff
@@ -1,4 +1,56 @@
+require "tilt/erubis"
+require "tilt"
+require "facets"
+require "ostruct"
+require "rack"
+
 module Hemp
-  class Controller
+  class BaseController
+    attr_reader :response
+
+    def render(action, instance_vars = get_instance_vars)
+      controller_folder = self.class.name.pathize.split("_").first
+      tilt_template = get_tilt_template(controller_folder, action)
+      instance_vars.delete_field :request
+      instance_vars.delete_field :params
+      set_response(tilt_template.render(instance_vars))
+    end
+
+    def get_tilt_template(controller_folder, action)
+      template_file = File.join(
+        "app", "views", controller_folder, "#{action}.erb"
+      )
+
+      Tilt.new template_file
+    end
+
+    def get_instance_vars
+      vars = self.instance_variables
+      instance_vars = {}
+
+      OpenStruct.new(pile_up_instance_vars vars, instance_vars)
```

Comment:
Add parentheses to nested method call `pile_up_instance_vars vars, instance_vars`.